core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -209,7 +209,7 @@ private FormValidation updateData(String json, boolean signatureCheck)
             }
         }
 
-        LOGGER.info("Obtained the latest update center data file for UpdateSource " + id);
+        LOGGER.finest("Obtained the latest update center data file for UpdateSource " + id);
```

Comment:
As above, would prefer `FINE`. (Also note that you should use either a message format or a lambda when logging a computed string at below `INFO`.)